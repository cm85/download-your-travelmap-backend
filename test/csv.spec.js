const csv = require('../app/csv');
const fs = require('fs');

const html = fs.readFileSync(`${__dirname}/fixtures/travelmap.html`, 'utf8');
const map = require('../app/map')(html);

test('creates csv', async () => {
  const actualCsv = await csv(map);
  expect(actualCsv).toMatchSnapshot();
});
