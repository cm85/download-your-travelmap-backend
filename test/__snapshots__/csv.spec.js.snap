// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates csv 1`] = `
"\\"lat\\",\\"lon\\",\\"name\\",\\"country\\",\\"city\\",\\"iso\\",\\"been\\"
40.96427,-5.66385,\\"Salamanca, Spain\\",\\"Spain\\",\\"Salamanca\\",\\"ES\\",\\"been\\"
51.51661,7.4583,\\"Dortmund, Germany\\",\\"Germany\\",\\"Dortmund\\",\\"DE\\",\\"been\\"
57.44707,10.530396,\\"Frederikshavn, Denmark\\",\\"Denmark\\",\\"Frederikshavn\\",\\"DK\\",\\"been\\"
50.939663,6.959839,\\"Cologne, Germany\\",\\"Germany\\",\\"Cologne\\",\\"DE\\",\\"been\\"
50.444885,30.536667,\\"Kiev, Ukraine\\",\\"Ukraine\\",\\"Kiev\\",\\"UA\\",\\"been\\"
-24.78866,-65.41018,\\"Salta, Argentina\\",\\"Argentina\\",\\"Salta\\",\\"AR\\",\\"been\\"
43.84202,10.51535,\\"Lucca, Italy\\",\\"Italy\\",\\"Lucca\\",\\"IT\\",\\"been\\"
51.891148,-8.474197,\\"Cork, Ireland\\",\\"Ireland\\",\\"Cork\\",\\"IE\\",\\"been\\"
43.549717,10.31293,\\"Livorno, Italy\\",\\"Italy\\",\\"Livorno\\",\\"IT\\",\\"been\\"
54.786343,9.4347,\\"Flensburg, Germany\\",\\"Germany\\",\\"Flensburg\\",\\"DE\\",\\"been\\"
48.049057,8.199312,\\"Furtwangen, Germany\\",\\"Germany\\",\\"Furtwangen\\",\\"DE\\",\\"been\\"
54.32192,10.134516,\\"Kiel, Germany\\",\\"Germany\\",\\"Kiel\\",\\"DE\\",\\"been\\"
43.767918,11.252379,\\"Florence, Italy\\",\\"Italy\\",\\"Florence\\",\\"IT\\",\\"been\\"
41.385597,2.169576,\\"Barcelona, Spain\\",\\"Spain\\",\\"Barcelona\\",\\"ES\\",\\"been\\"
38.71211,-9.138085,\\"Lisbon, Portugal\\",\\"Portugal\\",\\"Lisbon\\",\\"PT\\",\\"been\\"
43.2964,5.37,\\"Marseille, France\\",\\"France\\",\\"Marseille\\",\\"FR\\",\\"been\\"
40.07627,113.30001,\\"Datong, China\\",\\"China\\",\\"Datong\\",\\"CN\\",\\"been\\"
40.713238,-74.00584,\\"New York City, NY, USA\\",\\"NY\\",\\"New York City\\",\\"\\",\\"been\\"
39.473854,-0.377154,\\"Valencia, Spain\\",\\"Spain\\",\\"Valencia\\",\\"ES\\",\\"been\\"
37.508392,15.080838,\\"Catania, Italy\\",\\"Italy\\",\\"Catania\\",\\"IT\\",\\"been\\"
53.27695,6.15293,\\"Kollum, The Netherlands\\",\\"The Netherlands\\",\\"Kollum\\",\\"NL\\",\\"been\\"
-29.90267,-71.25127,\\"La Serena, Chile\\",\\"Chile\\",\\"La Serena\\",\\"CL\\",\\"been\\"
42.46519,1.89037,\\"Latour-de-Carol, France\\",\\"France\\",\\"Latour-de-Carol\\",\\"FR\\",\\"been\\"
52.22875,21.00625,\\"Warsaw, Poland\\",\\"Poland\\",\\"Warsaw\\",\\"PL\\",\\"been\\"
50.95326,1.854357,\\"Calais, France\\",\\"France\\",\\"Calais\\",\\"FR\\",\\"been\\"
54.087986,12.13723,\\"Rostock, Germany\\",\\"Germany\\",\\"Rostock\\",\\"DE\\",\\"been\\"
45.64233,25.58853,\\"Brasov, Romania\\",\\"Romania\\",\\"Brasov\\",\\"RO\\",\\"been\\"
54.972885,-1.618424,\\"Newcastle upon Tyne, UK\\",\\"UK\\",\\"Newcastle upon Tyne\\",\\"GB\\",\\"been\\"
51.73223,3.77436,\\"Renesse, The Netherlands\\",\\"The Netherlands\\",\\"Renesse\\",\\"NL\\",\\"been\\"
53.324997,5.998067,\\"Dokkum, The Netherlands\\",\\"The Netherlands\\",\\"Dokkum\\",\\"NL\\",\\"been\\"
-39.417824,175.39963,\\"Ohakune, New Zealand\\",\\"New Zealand\\",\\"Ohakune\\",\\"NZ\\",\\"been\\"
44.17714,28.65308,\\"Constanta, Romania\\",\\"Romania\\",\\"Constanta\\",\\"RO\\",\\"been\\"
-43.46552,170.01831,\\"Fox Glacier, New Zealand\\",\\"New Zealand\\",\\"Fox Glacier\\",\\"NZ\\",\\"been\\"
50.036472,19.25247,\\"Oswiecim, Poland\\",\\"Poland\\",\\"Oswiecim\\",\\"PL\\",\\"been\\"
-33.918163,18.424736,\\"Cape Town Central, South Africa\\",\\"South Africa\\",\\"Cape Town Central\\",\\"ZA\\",\\"want\\"
51.344887,12.386192,\\"Leipzig, Germany\\",\\"Germany\\",\\"Leipzig\\",\\"DE\\",\\"been\\"
40.922207,9.5003,\\"Olbia, Italy\\",\\"Italy\\",\\"Olbia\\",\\"IT\\",\\"been\\"
55.67502,12.580593,\\"Copenhagen, Denmark\\",\\"Denmark\\",\\"Copenhagen\\",\\"DK\\",\\"been\\"
-29.9604,-71.33852,\\"Coquimbo, Chile\\",\\"Chile\\",\\"Coquimbo\\",\\"CL\\",\\"been\\"
-33.895325,151.26826,\\"Bondi, Australia\\",\\"Australia\\",\\"Bondi\\",\\"AU\\",\\"been\\"
40.41471,-3.700708,\\"Madrid, Spain\\",\\"Spain\\",\\"Madrid\\",\\"ES\\",\\"been\\"
40.378716,9.694828,\\"Orosei, Italy\\",\\"Italy\\",\\"Orosei\\",\\"IT\\",\\"been\\"
55.43124,13.82827,\\"Ystad, Sweden\\",\\"Sweden\\",\\"Ystad\\",\\"SE\\",\\"been\\"
60.16798,24.940203,\\"Helsinki, Finland\\",\\"Finland\\",\\"Helsinki\\",\\"FI\\",\\"been\\"
41.49958,-81.69382,\\"Cleveland, OH, USA\\",\\"OH\\",\\"Cleveland\\",\\"\\",\\"been\\"
-33.02397,-71.5522,\\"Vina del Mar, Chile\\",\\"Chile\\",\\"Vina del Mar\\",\\"CL\\",\\"been\\"
44.42677,26.103397,\\"Bucharest, Romania\\",\\"Romania\\",\\"Bucharest\\",\\"RO\\",\\"been\\"
31.234953,121.48737,\\"Shanghai, China\\",\\"China\\",\\"Shanghai\\",\\"CN\\",\\"been\\"
44.652786,-1.179532,\\"Arcachon, France\\",\\"France\\",\\"Arcachon\\",\\"FR\\",\\"been\\"
43.606926,3.880531,\\"Montpellier, France\\",\\"France\\",\\"Montpellier\\",\\"FR\\",\\"been\\"
41.893623,12.495978,\\"Rome, Italy\\",\\"Italy\\",\\"Rome\\",\\"IT\\",\\"been\\"
43.86231,18.432571,\\"Sarajevo, Bosnia and Herzegovina\\",\\"Bosnia and Herzegovina\\",\\"Sarajevo\\",\\"BA\\",\\"been\\"
31.26269,121.07674,\\"Suzhou, China\\",\\"China\\",\\"Suzhou\\",\\"CN\\",\\"been\\"
54.18147,7.89033,\\"Helgoland, Germany\\",\\"Germany\\",\\"Helgoland\\",\\"DE\\",\\"been\\"
43.3107,10.523853,\\"Cecina, Italy\\",\\"Italy\\",\\"Cecina\\",\\"IT\\",\\"been\\"
43.34368,17.80821,\\"Mostar, Bosnia and Herzegovina\\",\\"Bosnia and Herzegovina\\",\\"Mostar\\",\\"BA\\",\\"been\\"
39.909336,116.39452,\\"Beijing, China\\",\\"China\\",\\"Beijing\\",\\"CN\\",\\"been\\"
48.369976,10.88333,\\"Augsburg, Germany\\",\\"Germany\\",\\"Augsburg\\",\\"DE\\",\\"been\\"
50.849632,4.351959,\\"Brussels, Belgium\\",\\"Belgium\\",\\"Brussels\\",\\"BE\\",\\"been\\"
43.7383,7.42696,\\"Monte-Carlo, Monaco\\",\\"Monaco\\",\\"Monte-Carlo\\",\\"MC\\",\\"been\\"
49.446163,11.07039,\\"Nuremberg, Germany\\",\\"Germany\\",\\"Nuremberg\\",\\"DE\\",\\"been\\"
59.330135,18.06151,\\"Stockholm, Sweden\\",\\"Sweden\\",\\"Stockholm\\",\\"SE\\",\\"been\\"
51.050453,13.73823,\\"Dresden, Germany\\",\\"Germany\\",\\"Dresden\\",\\"DE\\",\\"been\\"
41.902363,12.454423,\\"Vatican City, Vatican City\\",\\"Vatican City\\",\\"Vatican City\\",\\"VA\\",\\"been\\"
48.13641,11.57753,\\"Munich, Germany\\",\\"Germany\\",\\"Munich\\",\\"DE\\",\\"been\\"
56.87996,14.79945,\\"Vaxjo, Sweden\\",\\"Sweden\\",\\"Vaxjo\\",\\"SE\\",\\"been\\"
-42.450237,171.20314,\\"Greymouth, New Zealand\\",\\"New Zealand\\",\\"Greymouth\\",\\"NZ\\",\\"been\\"
39.525715,2.73611,\\"Playa de Palma, Spain\\",\\"Spain\\",\\"Playa de Palma\\",\\"ES\\",\\"been\\"
56.66613,16.36395,\\"Kalmar, Sweden\\",\\"Sweden\\",\\"Kalmar\\",\\"SE\\",\\"been\\"
53.406662,-2.974251,\\"Liverpool, UK\\",\\"UK\\",\\"Liverpool\\",\\"GB\\",\\"been\\"
-42.40858,173.67859,\\"Kaikoura, New Zealand\\",\\"New Zealand\\",\\"Kaikoura\\",\\"NZ\\",\\"been\\"
51.51924,-0.096654,\\"London, UK\\",\\"UK\\",\\"London\\",\\"GB\\",\\"been\\"
51.84828,104.87136,\\"Listvyanka, Russia\\",\\"Russia\\",\\"Listvyanka\\",\\"RU\\",\\"been\\"
59.43641,24.7526,\\"Tallinn, Estonia\\",\\"Estonia\\",\\"Tallinn\\",\\"EE\\",\\"been\\"
-37.63952,176.18474,\\"Mount Maunganui, New Zealand\\",\\"New Zealand\\",\\"Mount Maunganui\\",\\"NZ\\",\\"been\\"
51.35729,11.09631,\\"Bad Frankenhausen, Germany\\",\\"Germany\\",\\"Bad Frankenhausen\\",\\"DE\\",\\"been\\"
57.66748,15.85453,\\"Vimmerby, Sweden\\",\\"Sweden\\",\\"Vimmerby\\",\\"SE\\",\\"been\\"
52.29778,104.29639,\\"Irkutsk, Russia\\",\\"Russia\\",\\"Irkutsk\\",\\"RU\\",\\"been\\"
42.6999,9.44988,\\"Bastia, France\\",\\"France\\",\\"Bastia\\",\\"FR\\",\\"been\\"
42.763542,10.880054,\\"Castiglione Della Pescaia, Italy\\",\\"Italy\\",\\"Castiglione Della Pescaia\\",\\"IT\\",\\"been\\"
51.96149,7.622354,\\"Muenster, Germany\\",\\"Germany\\",\\"Muenster\\",\\"DE\\",\\"been\\"
52.37,4.89,\\"Amsterdam, The Netherlands\\",\\"The Netherlands\\",\\"Amsterdam\\",\\"NL\\",\\"been\\"
48.857037,2.349401,\\"Paris, France\\",\\"France\\",\\"Paris\\",\\"FR\\",\\"been\\"
52.076355,4.310603,\\"The Hague, The Netherlands\\",\\"The Netherlands\\",\\"The Hague\\",\\"NL\\",\\"been\\"
40.840416,14.252355,\\"Naples, Italy\\",\\"Italy\\",\\"Naples\\",\\"IT\\",\\"been\\"
51.920525,4.480866,\\"Rotterdam, The Netherlands\\",\\"The Netherlands\\",\\"Rotterdam\\",\\"NL\\",\\"been\\"
50.802788,-1.086526,\\"Portsmouth, UK\\",\\"UK\\",\\"Portsmouth\\",\\"GB\\",\\"been\\"
-41.28883,174.00539,\\"Picton, New Zealand\\",\\"New Zealand\\",\\"Picton\\",\\"NZ\\",\\"been\\"
47.374664,8.539639,\\"Zurich, Switzerland\\",\\"Switzerland\\",\\"Zurich\\",\\"CH\\",\\"been\\"
55.751865,37.622437,\\"Moscow, Russia\\",\\"Russia\\",\\"Moscow\\",\\"RU\\",\\"been\\"
45.755955,4.832282,\\"Lyon, France\\",\\"France\\",\\"Lyon\\",\\"FR\\",\\"been\\"
42.653934,18.079966,\\"Dubrovnik, Croatia\\",\\"Croatia\\",\\"Dubrovnik\\",\\"HR\\",\\"been\\"
-43.803658,172.96826,\\"Akaroa, New Zealand\\",\\"New Zealand\\",\\"Akaroa\\",\\"NZ\\",\\"been\\"
51.76587,10.65524,\\"Schierke, Germany\\",\\"Germany\\",\\"Schierke\\",\\"DE\\",\\"been\\"
53.345966,-6.25341,\\"Dublin, Ireland\\",\\"Ireland\\",\\"Dublin\\",\\"IE\\",\\"been\\"
-36.88819,175.8202,\\"Hot Water Beach, New Zealand\\",\\"New Zealand\\",\\"Hot Water Beach\\",\\"NZ\\",\\"been\\"
55.597195,12.998863,\\"Malmo, Sweden\\",\\"Sweden\\",\\"Malmo\\",\\"SE\\",\\"been\\"
43.71651,10.400349,\\"Pisa, Italy\\",\\"Italy\\",\\"Pisa\\",\\"IT\\",\\"been\\"
56.9619,23.72264,\\"Jurmala, Latvia\\",\\"Latvia\\",\\"Jurmala\\",\\"LV\\",\\"been\\"
53.27815,-9.050388,\\"Galway, Ireland\\",\\"Ireland\\",\\"Galway\\",\\"IE\\",\\"been\\"
-22.910463,-68.19538,\\"San Pedro de Atacama, Chile\\",\\"Chile\\",\\"San Pedro de Atacama\\",\\"CL\\",\\"been\\"
56.94834,24.107779,\\"Riga, Latvia\\",\\"Latvia\\",\\"Riga\\",\\"LV\\",\\"been\\"
54.99243,-7.31921,\\"Derry, UK\\",\\"UK\\",\\"Derry\\",\\"GB\\",\\"been\\"
43.603947,1.44303,\\"Toulouse, France\\",\\"France\\",\\"Toulouse\\",\\"FR\\",\\"been\\"
-34.58881,-58.39562,\\"Buenos Aires, Argentina\\",\\"Argentina\\",\\"Buenos Aires\\",\\"AR\\",\\"been\\"
48.781322,9.183353,\\"Stuttgart, Germany\\",\\"Germany\\",\\"Stuttgart\\",\\"DE\\",\\"been\\"
41.878407,-87.62568,\\"Chicago, IL, USA\\",\\"IL\\",\\"Chicago\\",\\"\\",\\"been\\"
60.38918,5.331452,\\"Bergen, Norway\\",\\"Norway\\",\\"Bergen\\",\\"NO\\",\\"been\\"
49.48651,8.46678,\\"Mannheim, Germany\\",\\"Germany\\",\\"Mannheim\\",\\"DE\\",\\"been\\"
43.317516,-1.975921,\\"San Sebastian - Donostia, Spain\\",\\"Spain\\",\\"San Sebastian - Donostia\\",\\"ES\\",\\"been\\"
52.48399,-1.888948,\\"Birmingham, UK\\",\\"UK\\",\\"Birmingham\\",\\"GB\\",\\"been\\"
37.105007,-8.67513,\\"Lagos, Portugal\\",\\"Portugal\\",\\"Lagos\\",\\"PT\\",\\"been\\"
50.636906,3.065442,\\"Lille, France\\",\\"France\\",\\"Lille\\",\\"FR\\",\\"been\\"
43.262074,-2.931721,\\"Bilbao, Spain\\",\\"Spain\\",\\"Bilbao\\",\\"ES\\",\\"been\\"
43.947704,4.806896,\\"Avignon, France\\",\\"France\\",\\"Avignon\\",\\"FR\\",\\"been\\"
52.39935,13.04793,\\"Potsdam, Germany\\",\\"Germany\\",\\"Potsdam\\",\\"DE\\",\\"been\\"
53.55556,10.013171,\\"Hamburg, Germany\\",\\"Germany\\",\\"Hamburg\\",\\"DE\\",\\"been\\"
-20.269703,148.71977,\\"Airlie Beach, Australia\\",\\"Australia\\",\\"Airlie Beach\\",\\"AU\\",\\"been\\"
-38.136536,176.25145,\\"Rotorua, New Zealand\\",\\"New Zealand\\",\\"Rotorua\\",\\"NZ\\",\\"been\\"
-38.689762,176.07594,\\"Taupo, New Zealand\\",\\"New Zealand\\",\\"Taupo\\",\\"NZ\\",\\"been\\"
41.00986,28.95707,\\"Istanbul, Turkey\\",\\"Turkey\\",\\"Istanbul\\",\\"TR\\",\\"been\\"
-33.43771,-70.64779,\\"Santiago, Chile\\",\\"Chile\\",\\"Santiago\\",\\"CL\\",\\"been\\"
-14.46397,132.26505,\\"Katherine, Australia\\",\\"Australia\\",\\"Katherine\\",\\"AU\\",\\"been\\"
-41.286087,174.77695,\\"Wellington, New Zealand\\",\\"New Zealand\\",\\"Wellington\\",\\"NZ\\",\\"been\\"
-33.04531,-71.62038,\\"Valparaiso, Chile\\",\\"Chile\\",\\"Valparaiso\\",\\"CL\\",\\"been\\"
-41.51191,173.95515,\\"Blenheim, New Zealand\\",\\"New Zealand\\",\\"Blenheim\\",\\"NZ\\",\\"been\\"
-43.532024,172.63718,\\"Christchurch, New Zealand\\",\\"New Zealand\\",\\"Christchurch\\",\\"NZ\\",\\"been\\"
54.59679,-5.930921,\\"Belfast, UK\\",\\"UK\\",\\"Belfast\\",\\"GB\\",\\"been\\"
54.69059,25.26981,\\"Vilnius, Lithuania\\",\\"Lithuania\\",\\"Vilnius\\",\\"LT\\",\\"been\\"
47.998714,7.851038,\\"Freiburg im Breisgau, Germany\\",\\"Germany\\",\\"Freiburg im Breisgau\\",\\"DE\\",\\"been\\"
49.29524,19.95312,\\"Zakopane, Poland\\",\\"Poland\\",\\"Zakopane\\",\\"PL\\",\\"been\\"
54.70905,20.50595,\\"Kaliningrad, Russia\\",\\"Russia\\",\\"Kaliningrad\\",\\"RU\\",\\"been\\"
59.91173,10.750844,\\"Oslo, Norway\\",\\"Norway\\",\\"Oslo\\",\\"NO\\",\\"been\\"
-42.52297,172.82918,\\"Hanmer Springs, New Zealand\\",\\"New Zealand\\",\\"Hanmer Springs\\",\\"NZ\\",\\"been\\"
53.083763,8.819797,\\"Bremen, Germany\\",\\"Germany\\",\\"Bremen\\",\\"DE\\",\\"been\\"
58.75326,17.00923,\\"Nykoping, Sweden\\",\\"Sweden\\",\\"Nykoping\\",\\"SE\\",\\"been\\"
37.778934,-122.41579,\\"San Francisco, CA, USA\\",\\"CA\\",\\"San Francisco\\",\\"\\",\\"been\\"
52.51959,13.397209,\\"Berlin, Germany\\",\\"Germany\\",\\"Berlin\\",\\"DE\\",\\"been\\"
58.710266,13.822889,\\"Mariestad, Sweden\\",\\"Sweden\\",\\"Mariestad\\",\\"SE\\",\\"been\\"
-34.45451,-57.83755,\\"Colonia del Sacramento, Uruguay\\",\\"Uruguay\\",\\"Colonia del Sacramento\\",\\"UY\\",\\"been\\"
37.383396,-5.990674,\\"Seville, Spain\\",\\"Spain\\",\\"Seville\\",\\"ES\\",\\"been\\"
49.40902,8.697485,\\"Heidelberg, Germany\\",\\"Germany\\",\\"Heidelberg\\",\\"DE\\",\\"been\\"
35.680565,139.759,\\"Tokyo, Japan\\",\\"Japan\\",\\"Tokyo\\",\\"JP\\",\\"been\\"
57.72481,10.59614,\\"Skagen, Denmark\\",\\"Denmark\\",\\"Skagen\\",\\"DK\\",\\"been\\"
-45.03106,168.66235,\\"Queenstown, New Zealand\\",\\"New Zealand\\",\\"Queenstown\\",\\"NZ\\",\\"been\\"
-45.404236,167.71991,\\"Te Anau, New Zealand\\",\\"New Zealand\\",\\"Te Anau\\",\\"NZ\\",\\"been\\"
50.076847,14.427817,\\"Prague, Czech Republic\\",\\"Czech Republic\\",\\"Prague\\",\\"CZ\\",\\"been\\"
-44.005653,170.4839,\\"Lake Tekapo, New Zealand\\",\\"New Zealand\\",\\"Lake Tekapo\\",\\"NZ\\",\\"been\\"
48.583153,7.7636,\\"Strasbourg, France\\",\\"France\\",\\"Strasbourg\\",\\"FR\\",\\"been\\"
52.371662,9.735982,\\"Hannover, Germany\\",\\"Germany\\",\\"Hannover\\",\\"DE\\",\\"been,fave\\"
-31.40716,-64.18567,\\"Cordoba, Argentina\\",\\"Argentina\\",\\"Cordoba\\",\\"AR\\",\\"been\\"
43.5768,10.56142,\\"Crespina, Italy\\",\\"Italy\\",\\"Crespina\\",\\"IT\\",\\"been\\"
40.6347,17.94023,\\"Brindisi, Italy\\",\\"Italy\\",\\"Brindisi\\",\\"IT\\",\\"been\\"
47.56868,7.587512,\\"Basel, Switzerland\\",\\"Switzerland\\",\\"Basel\\",\\"CH\\",\\"been\\"
44.841103,-0.580674,\\"Bordeaux, France\\",\\"France\\",\\"Bordeaux\\",\\"FR\\",\\"been\\"
-44.697247,169.13551,\\"Wanaka, New Zealand\\",\\"New Zealand\\",\\"Wanaka\\",\\"NZ\\",\\"been\\"
-43.38766,170.18181,\\"Franz Josef, New Zealand\\",\\"New Zealand\\",\\"Franz Josef\\",\\"NZ\\",\\"been\\"
43.700638,7.276729,\\"Nice, France\\",\\"France\\",\\"Nice\\",\\"FR\\",\\"been\\"
55.46529,8.45692,\\"Esbjerg, Denmark\\",\\"Denmark\\",\\"Esbjerg\\",\\"DK\\",\\"been\\"
57.70382,11.972486,\\"Gothenburg, Sweden\\",\\"Sweden\\",\\"Gothenburg\\",\\"SE\\",\\"been\\"
45.439087,12.323647,\\"Venice, Italy\\",\\"Italy\\",\\"Venice\\",\\"IT\\",\\"been\\"
35.454258,139.56802,\\"Yokohama, Japan\\",\\"Japan\\",\\"Yokohama\\",\\"JP\\",\\"been\\"
53.47724,6.16124,\\"Schiermonnikoog, The Netherlands\\",\\"The Netherlands\\",\\"Schiermonnikoog\\",\\"NL\\",\\"been\\"
41.116333,1.248222,\\"Tarragona, Spain\\",\\"Spain\\",\\"Tarragona\\",\\"ES\\",\\"been\\"
60.83897,7.12124,\\"Flam, Norway\\",\\"Norway\\",\\"Flam\\",\\"NO\\",\\"been\\"
54.56544,8.54807,\\"Hallig Hooge, Germany\\",\\"Germany\\",\\"Hallig Hooge\\",\\"DE\\",\\"been\\"
47.92211,106.91698,\\"Ulaanbaatar, Mongolia\\",\\"Mongolia\\",\\"Ulaanbaatar\\",\\"MN\\",\\"been\\"
48.208862,16.372383,\\"Vienna, Austria\\",\\"Austria\\",\\"Vienna\\",\\"AT\\",\\"been\\"
43.628094,10.29188,\\"Tirrenia, Italy\\",\\"Italy\\",\\"Tirrenia\\",\\"IT\\",\\"been\\"
50.06046,19.9324,\\"Krakow, Poland\\",\\"Poland\\",\\"Krakow\\",\\"PL\\",\\"been\\"
53.476948,-2.247661,\\"Manchester, UK\\",\\"UK\\",\\"Manchester\\",\\"GB\\",\\"been\\"
38.246235,21.735107,\\"Patras, Greece\\",\\"Greece\\",\\"Patras\\",\\"GR\\",\\"been\\"
37.97698,23.721302,\\"Athens, Greece\\",\\"Greece\\",\\"Athens\\",\\"GR\\",\\"been\\"
-33.870037,151.20955,\\"Sydney, Australia\\",\\"Australia\\",\\"Sydney\\",\\"AU\\",\\"been\\"
-23.70267,133.88133,\\"Alice Springs, Australia\\",\\"Australia\\",\\"Alice Springs\\",\\"AU\\",\\"been\\"
-12.460347,130.84166,\\"Darwin, Australia\\",\\"Australia\\",\\"Darwin\\",\\"AU\\",\\"been\\"
57.58847,9.96,\\"Hirtshals, Denmark\\",\\"Denmark\\",\\"Hirtshals\\",\\"DK\\",\\"been\\"
48.149662,17.108503,\\"Bratislava, Slovakia\\",\\"Slovakia\\",\\"Bratislava\\",\\"SK\\",\\"been\\"
-27.4681,153.02197,\\"Brisbane, Australia\\",\\"Australia\\",\\"Brisbane\\",\\"AU\\",\\"been\\"
-31.955471,115.86148,\\"Perth, Australia\\",\\"Australia\\",\\"Perth\\",\\"AU\\",\\"been\\"
30.27296,120.15567,\\"Hangzhou, China\\",\\"China\\",\\"Hangzhou\\",\\"CN\\",\\"been\\"
34.05106,-118.23284,\\"Los Angeles, CA, USA\\",\\"CA\\",\\"Los Angeles\\",\\"\\",\\"been\\"
47.49346,19.0507,\\"Budapest, Hungary\\",\\"Hungary\\",\\"Budapest\\",\\"HU\\",\\"been\\"
-36.849407,174.76103,\\"Auckland Central, New Zealand\\",\\"New Zealand\\",\\"Auckland Central\\",\\"NZ\\",\\"been\\"
53.24759,10.41414,\\"Luneburg, Germany\\",\\"Germany\\",\\"Luneburg\\",\\"DE\\",\\"been\\""
`;
